cfy_manager/main.py

```diff
@@ -388,6 +405,8 @@ def db_node_add(**kwargs):
 
 
 @argh.named('remove')
+@argh.arg('-c', '--config-file', action='append', default=None,
```

Comment:
Perhaps this should have the default, rather than it living in the function?

Reply:
I tried.  If you combine default value with append action, then the default value always ends up in the parameter

Reply:
Ah, that's a pain- probably because of the multiple places to put the arg, but oh well!